pkg/security/secl/model/model_unix.go

```diff
@@ -195,6 +195,8 @@ type Event struct {
 	NetDevice        NetDeviceEvent        `field:"-" json:"-"`
 	VethPair         VethPairEvent         `field:"-" json:"-"`
 	UnshareMountNS   UnshareMountNSEvent   `field:"-" json:"-"`
+	// used for ebpfless
+	ClientID uint64 `field:"-" json:"-"`
```

Comment:
nit: is it used ?

Reply:
Yep it's used in field_handlers_ebpfless to call the process resolver